src/components/Header.js

```diff
@@ -248,27 +203,15 @@ export default function Header({ githubUrl, inverse, framework, firstChapter, is
                       <Meta>
                         <Title>Vue</Title>
                         <Detail>
-                          <LanguageLink
-                            className="tertiary"
-                            isGatsby
-                            to={`/vue/en/${firstChapter}/`}
-                          >
-                            English
-                          </LanguageLink>
-                          <LanguageLink
-                            className="tertiary"
-                            isGatsby
-                            to={`/vue/pt/${firstChapter}/`}
-                          >
-                            Português
-                          </LanguageLink>
+                          <LanguageLink to={`/vue/en/${firstChapter}/`}>English</LanguageLink>
+                          <LanguageLink to={`/vue/pt/${firstChapter}/`}>Português</LanguageLink>
                         </Detail>
                       </Meta>
                     </TooltipItem>
                   </TooltipList>
                 }
               >
-                <NavLink className={inverse ? 'inverse' : 'tertiary'}>
+                <NavLink {...(inverse ? { inverse: true } : { tertiary: true })}>
```

Comment:
Would this read better as `<NavLink inverse={inverse} tertiary={!inverse}>`?

Reply:
Yeah good call. That's cleaner. I'll update.